src/App.js

```diff
@@ -147,4 +53,12 @@ class App extends React.Component {
     }
 }
 
-export default App;
\ No newline at end of file
+const mapDispatchToProps = dispatch => ({
+    authorization: bindActionCreators(authorization, dispatch)
+});
```

Comment:
i don't think you need it
https://react-redux.js.org/using-react-redux/connect-mapdispatch#defining-mapdispatchtoprops-as-an-object

Reply:
you're right
I will fix this